app/views/shared/_navbar.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="navbar navbar-expand-sm navbar-light navbar-lewagon">
+  <%= link_to "#", class: "navbar-brand" do %>
+    <%= image_tag "https://raw.githubusercontent.com/lewagon/fullstack-images/master/uikit/logo.png" %>
```

Comment:
Il faudra remplacer par un logo qu'on fait/trouve

Reply:
Oui, tout à fait. Il me semble que c'est juste à premier jet sans custom! A intégrer dans la to do list d’aujourd’hui !

Reply:
ok !! on le merge du coup avant notre pull du coup ?